app/views/partials/language_selection.scala.html

```diff
@@ -0,0 +1,65 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import langswitch.Link
+@import langswitch.Language
+@import langswitch.Languages
+@import requests.RequestSupport
+@import play.api.mvc.RequestHeader
+@import play.twirl.api.Html
+
+
+@*
+This is a `language-selection` (copied from play-lang)
```

Comment:
Cant we inject this in?

Reply:
Yes, it's injectable. It's injected into `Layout.scala.html`. I'm not sure if I answered what you were asking for :/